tests/test_cli.py

```diff
@@ -86,6 +86,16 @@
     'workdir': 'test-workdir'
 }
 
+SKIPPER_CONF_WITH_GIT_REV = {
+    'registry': REGISTRY,
+    'build-container-image': SKIPPER_CONF_BUILD_CONTAINER_IMAGE,
+    'build-container-tag': SKIPPER_CONF_BUILD_CONTAINER_TAG,
+    'make': {
+        'makefile': SKIPPER_CONF_MAKEFILE,
+    },
+    'use_git_revision': 'true'
```

Comment:
Perhaps use a name that better represent that parameter is relevant for the build container tag.